miner/worker.go

```diff
@@ -1196,8 +1196,42 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 		if w.commitBundle(bundleTxs, w.coinbase, interrupt) {
 			return
 		}
-		w.current.profit.Add(w.current.profit, bundle.totalEth)
+		w.current.profit.Add(w.current.profit, bundle.ethSentToCoinbase)
```

Comment:
Just curious - why has the profic calculation here changed from `totalEth` to `ethSentToCoinbase`?